content/communities/crowdsourcing-and-citizen-science.md

```diff
@@ -17,6 +17,7 @@ topics:
 authors:
   - jay-benforado
   - sophia-b-liu
+  -john-mcLaughlin
```

Comment:
Can we check that `john-mcLaughlin` is in our system? And can we add a space after the dash?

Reply:
He's not in the system https://digital.gov/authors/ so we'll need to add him before merging this.

Reply:
I added him